src/schema/v2/order/types/DeliveryInfo.ts

```diff
@@ -0,0 +1,63 @@
+import { GraphQLFieldConfig, GraphQLObjectType, GraphQLString } from "graphql"
+import { OrderJSON } from "./exchangeJson"
+import { ResolverContext } from "types/graphql"
+
+const DeliveryInfoType = new GraphQLObjectType({
+  name: "DeliveryInfo",
+  description: "Shipment details for an order",
+  fields: () => ({
+    shipperName: {
+      type: GraphQLString,
+      description:
+        "The carrier handling the shipment as saved on the order (e.g., UPS, FedEx, DHL, USPS)",
+    },
+    trackingNumber: {
+      type: GraphQLString,
+      description: "The tracking number for the shipment",
+    },
+    trackingURL: {
+      type: GraphQLString,
+      description: "The URL to track the shipment",
+    },
+
+    estimatedDelivery: {
+      type: GraphQLString,
+      description: "Eesimated delivery date as saved on the order",
+    },
+
+    estimatedDeliveryWindow: {
+      type: GraphQLString,
+      description: "Eesimated delivery window as saved on the order",
+    },
```

Comment:
Estimated*

If one of these is an iso 8601 or similar date string it would be good to put in the descriptions field.